### ISSUE TITLE
Баг со смертью после долгого падения в воде

### ISSUE THREAD

**Issue by Xdem14** [Sokol-Games-GmbH/PixelCombats.PublicApi#170]
https://github.com/Sokol-Games-GmbH/PixelCombats.PublicApi/assets/151731809/6215725d-ae56-4b04-8d5e-ea77d58a82a5

**Comment by Evgene82** [comment]
Это специально так сделано, когда у игрока 7-10 секунд нет каллизии с блоками он умирает

**Comment by forwork10** [comment]
Да не должно там убивать... В воде если находится игрок

**Comment by justqstn** [comment]
каллизия

> Это специально так сделано, когда у игрока 7-10 секунд нет каллизии с блоками он умирает

**Comment by forwork10** [comment]
> Что не так? Все верно, каллизия, с водой не учитывается
> Специально сделали чтобы игрок когда в бездну упал не был вечно жив

Так, ну вообще мы задумывали чтобы такого небыло... И коллизия тут при чём? Тут уже у нас скрипты фигово работают... Поправим потом

**Comment by kkohno** [comment]
Выше верно описали - игрок какбы падает в течении 10 сек и он умирает. чтобы вечно в бездну не падал... Можем конечно в воде сделать исключение, но может быть карта с вобой огромная... Ваши предложения готовы выслушать

**Comment by Evgene82** [comment]
Спт всосал

**Comment by forwork10** [comment]
> Спт всосал

Так ало, баг такой не задумывали, как я и сказал)

**Comment by forwork10** [comment]
> Бездарь, это не баг

Именно баг. Ведь должно считать что игрок находится на блоке, тоесть в воде. А не прав в этой ситуации ты кстати, раз ты так не уверен в себе, что начал применять оскорбления в мою сторону...)

**Comment by Evgene82** [comment]
Ты не знаешь чё такое коллизия или что? Коллизия это прикосновение и касание с блоком игрока, а вода не касается игрока, это пустой блок, через него можно ходить
Также с решеткой и тд

**Comment by forwork10** [comment]
> Ты не знаешь чё такое коллизия или что? Коллизия это прикосновение и касание с блоком игрока, а вода не касается игрока, это пустой блок, через него можно ходить
> Также с решеткой и тд

Вижу совсем не знакомы с этим... Советую изучить работу коллайдеров в Unity...

**Comment by Evgene82** [comment]
Коллизия (Collision) - переводится как столкновение. Это очень широкий термин, в движках и визуализации он обозначает взаимодействие между объектами, а если точнее, то их столкновение (еще можно встретить понятие пересечение) и его результат.
